### PR TITLE
fix(react/vue): prevent usage of Cordova on vue and react projects

### PR DESCRIPTION
**Pull request by jcesarmobile**
Prevents users from adding Cordova to vue and react projects.

Prevents it for `ionic integrations enable cordova` and ` ionic cordova platform add`, not sure if more commands need it too.

### CODE REVIEW

File: packages/ionic/src/lib/integrations/cordova/index.ts [hunk 1]
```diff
@@ -21,6 +22,9 @@ export class Integration extends BaseIntegration<ProjectIntegration> {
   }
 
   async add(details: IntegrationAddDetails): Promise<void> {
+    if (this.e.project.type === 'vue' || this.e.project.type === 'angular') {
```
Comment (dwieeb) [inline packages/ionic/src/lib/integrations/cordova/index.ts]: Did you mean `react` here?

**Comment by brandyscarney** [comment]
It looks good! I think we should also link off to some documentation with the warning though.

File: packages/ionic/src/lib/integrations/cordova/index.ts (hunk 1 above)
Comment (jcesarmobile) [reply packages/ionic/src/lib/integrations/cordova/index.ts]: wops, fixed

**Comment by jcesarmobile** [comment]
@brandyscarney linked the overview pages that will contain the information once https://github.com/ionic-team/ionic-docs/pull/744 gets merged.

**Pull request merged**